code/game/turfs/simulated/floor/asteroid_floors.dm

```diff
@@ -81,19 +80,19 @@
 
 		to_chat(user, "<span class='notice'>You start digging...</span>")
 
-		playsound(src, I.usesound, 50, TRUE)
-		if(do_after(user, 40 * I.toolspeed, target = src))
+		playsound(src, used.usesound, 50, TRUE)
+		if(do_after(user, 40 * used.toolspeed, target = src))
```

Comment:
iirc there's a proc like `use_tool()` that'd work just as well here

Reply:
Probably but this change is invasive enough without fiddling with the underlying implementation